content/blog/2025-04-view-refactor.md

```diff
@@ -0,0 +1,65 @@
+---
+authors: ["kokkos-team"]
+title: "Kokkos::View meets std::mdspan"
+date: "2025-04-11"
+tags: ["2025", "View"]
+thumbnail: img/blog/2025/view-impl-bridge.jpg
+---
+
+After a long long time in the works, we finally merged the refactor of the
+`Kokkos::View` implementation details, which moves its underpinnings to
+leverage `std::mdspan` capabilities.
+
+This rework is essentially a complete rewrite of `Kokkos::View` throwing out
+thousands of lines of complicated code (lots of it dating back to before Kokkos
+was available on GitHub), and replacing them with something new.
+While the intent is that no user visible changes (with some exceptions noted below)
```

Comment:
```suggestion
While the intent is that no-user visible changes (with some exceptions noted below)
```

Reply:
why?

Reply:
Fixed the suggestion to `user-visible` as a compound.